ci/goreleaser.yml

```diff
@@ -0,0 +1,36 @@
+name: goreleaser
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+permissions:
+  contents: write
```

Comment:
should we add id-token here as well to encourage keyless signing? (we could add the cosign action too)

thoughts?

Reply:
added id-token